docs/eventing/accessing-traces.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Accessing CloudEvent Traces"
+#linkTitle: "OPTIONAL_ALTERNATE_NAV_TITLE"
+weight: 15
+type: "docs"
+---
+
+Depending on the request tracing tool that you have installed on your Knative
+Eventing cluster, see the corresponding section for details about how to
+visualize and trace your requests.
+
+## Installation
+
+Knative Eventing uses the same tracing plugin as Knative Serving. See the 
+[Tracing installation instructions](./../serving/installing-logging-metrics-traces.md#end-to-end-request-tracing)
+in the Knative Serving section for details. Note that you do not need to install the 
+Knative Serving component itself. To enable request tracing in Knative Eventing 
+you need to install only Elasticsearch and either the Zipkin or Jaeger plugins.
+
+## Configuration
+
+Most Knative Eventing tracing configuration is handled by the `config-tracing` ConfigMap in the `knative-eventing` namespace. This single ConfigMap controls tracing for:
+ - Brokers
+ - Triggers
+ - Channel Dispatchers
```

Comment:
I guess we haven't documented it. Logically Channels are broken into two components:
1. Controller - The Control plane work. E.g. modifying the status of K8s object, creating any other needed resources (such as Topics and Subscriptions in GCP PubSub).
1. Dispatecher - The Data plane work. E.g. events sent to the channel go here. Events are sent by the Channel from this piece to the Subscribers.

I hesitate adding it to the overview because it is an implementation detail that can be ignored until you want to debug or trace. So I prefer the overview to be based on the conceptual ideas of Channels, rather than their internal pieces. 

What if I just add an explanation inline here?